src/Build/BackEnd/BuildManager/BuildParameters.cs

```diff
@@ -801,6 +804,27 @@ public string OutputResultsCacheFile
             set => _outputResultsCacheFile = value;
         }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether file accesses should be reported to any configured project cache plugins.
+        /// </summary>
+        public bool ReportFileAccesses
+        {
+            get => _reportFileAccesses;
+            set
+            {
+                _reportFileAccesses = value;
+
+                if (_reportFileAccesses)
```

Comment:
I couldn't think of a good way to do that without introducing a bunch of nullables to disambiguate between explicit sets vs these defaultings. I'm open to ideas for how to best deal with this, if at all.

Reply:
I'm not 100% sure it actually makes sense to do that, since these are BuildParameters that users presumably don't use over and over again very often (if at all?). I think the normal pattern is to make a new BuildParameters per BuildRequest, but I wanted it at least considered 🙂